README.md

```diff
@@ -5,7 +5,7 @@
 <a href="https://packagist.org/packages/woocommerce/woocommerce"><img src="https://poser.pugx.org/woocommerce/woocommerce/v/stable" alt="Latest Stable Version"></a>
 <img src="https://img.shields.io/wordpress/plugin/dt/woocommerce.svg" alt="WordPress.org downloads">
 <img src="https://img.shields.io/wordpress/plugin/r/woocommerce.svg" alt="WordPress.org rating">
-<a href="https://travis-ci.com/woocommerce/woocommerce"><img src="https://travis-ci.com/woocommerce/woocommerce.svg?branch=trunk" alt="Build Status"></a>
+<a href="https://github.com/woocommerce/woocommerce/actions/workflows/ci.yml"><img src="https://github.com/woocommerce/woocommerce/actions/workflows/ci.yml/badge.svg?branch=trunk" alt="Build Status"></a>
```

Comment:
This looks weird. There's no `/actions` directory whatsoever (maybe you meant `/.github/workflows`?), there's no `badge.svg` file at all in the code, and I don't think that `ci.yml/badge.svg` would be the right path for it. Also, is it right to label a link to a static file as "Build status"?

![image](https://user-images.githubusercontent.com/937723/111434886-62119000-8700-11eb-8fd7-48a8f858166e.png)

Reply:
@Konamiman this will be only available once this PR gets merged into trunk, that's why is missing this badge.

Reply:
Here an example of our nightly builds:

https://github.com/woocommerce/woocommerce/actions/workflows/nightly-builds.yml
https://github.com/woocommerce/woocommerce/actions/workflows/nightly-builds.yml/badge.svg?branch=trunk


![Build Status](https://github.com/woocommerce/woocommerce/actions/workflows/nightly-builds.yml/badge.svg?branch=trunk)

Reply:
Ok, got it, thanks